orca/requester.py

```diff
@@ -66,12 +158,27 @@ def bisect_request(url, threshold=5e8):
         logger.debug(
             f"Splitting, request over threshold: {round(bytes * 10**-6)}/500 mb"
         )
-        start_end_format = re.compile(r"(([a-z]+)\[(\d*)(:\d*){0,1}:(\d*)\])")
-        var_time, var, start, stride, end = start_end_format.findall(url)[0]
+        start_end_format_full = re.compile(
```

Comment:
We're using `xarray` to open up the datasets. And it supports OPeNDAP. Have you considered whether, instead of manually doing all of the URL/OPeNDAP parsing ourselves (which could be fraught with problems), possibly we could just open it as an [xarray Dataset](https://docs.xarray.dev/en/stable/generated/xarray.Dataset.html#xarray.Dataset) and then utilize the object attributes (`coords` and `dims`) to get the information we need?